Cartfile

```diff
@@ -1 +1 @@
-github "LoopKit/LoopKit" "dev"
+github "tidepool-org/LoopKit" "basal-picker"
```

Comment:
What's your typical Cartfile strategy for merging changes that depend on different branches? Switch to the dependent branch while testing, then revert the Cartfile after merging?

Reply:
Yes. For me, I can build in a workspace setup, and not worry to much about updating carthage specs, but travis needs it updated, and if others check out the branch, it helps them be able to build.